packages/okta-auth-js/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-auth-js",
   "description": "The Okta Auth SDK",
-  "version": "3.1.2",
+  "version": "3.1.3",
```

Comment:
We had discussed an alpha/beta version for wider testing.  Are we confident that we've covered enough testing of the sequential calls?   (What is the allowed delay during a PKCE transaction, for example?)

Reply:
Since the commands will execute sequentially, it should work fine if one command takes a very long time, or fails. The next command will start after the first one finishes. For example, if 2 calls are made to getWithPopup and the user takes a long time to fill out the form. The first call may fail because of a timeout. But the challenge code for the 2nd popup flow will be computed after that failure, so it will start fresh.